packages/roosterjs-content-model-types/lib/contentModel/format/FormatHandlerTypeMap.ts

```diff
@@ -114,6 +115,11 @@ export interface FormatHandlerTypeMap {
      */
     id: IdFormat;
 
+    /**
+     * Format of ImageStateFormat
+     */
+    imageState: ImageStateFormat;
```

Comment:
@JiuqingSong could this be considered as breaking change?

Reply:
technically it is. But it should be fine because we don't expect developers create object of this type directly.

Reply:
I see, thanks.